src/components/process-video.ts

```
@@ -0,0 +1,456 @@
+        this.mediaRecorder = null;
+        this.recordedChunks = [];
+        this.previewStream = null;
+
+    } catch (error) {
+        console.error('Error stopping video recording:', error);
+    }
+  }
+
+  private handleFileUpload(event: Event) {
+    const file = (event.target as HTMLInputElement).files?.[0];
+    if (file && file.type.startsWith('video/')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)